pkg/image/apis/image/types.go

```diff
@@ -28,6 +28,16 @@ const (
 	// ExcludeImageSecretAnnotation indicates that a secret should not be returned by imagestream/secrets.
 	ExcludeImageSecretAnnotation = "openshift.io/image.excludeSecret"
 
+	// DefaultArchAnnotation represents an architecture that should be
+	// selected if an image uses a manifest list and it should be
+	// downconverted.
+	DefaultArchAnnotation = "openshift.io/image.defaultArch"
```

Comment:
Inside your own group, I'd expect this to be scoped to your group rather than claiming a bit of the global namespace.

`image.openshift.io/defaultArch`